app/facades/authors/show_facade.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+module Authors
+  class ShowFacade
+    attr_reader :author
+
+    def initialize(author)
+      @author = author
+    end
+
+    def author_name
+      @author.full_name
```

Comment:
delegate :full_name, :date_of_birth, :biography, :books, :avatar, to: :author, prefix: true

Reply:
https://itnext.io/understanding-delegate-in-ruby-on-rails-i-wish-i-knew-before-5edd341bad47